### ISSUE TITLE
Keep selected connection selected on return to Connect screen

### ISSUE THREAD

**Issue by mkromberg** [Dyalog/ride#369]
### Describe the issue you are having
If I start APL and then exit, RIDE brings the connect screen back up so I can start again, which is great. But my selection has been lost, the first connection is now selected.

### Did you connect to an already running interpreter or start the interpreter from RIDE?
Start an interpreter

### How do you reproduce the issue?
Start any connection other than the first one in the list. Type )OFF. The first connection is selected.

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: unknown
  Platform: Win32
  Date: unknown
  Git commit: unknown
  Preferences:{
    "colourScheme":"Francisco Goya",
    "colourSchemes":"[{\"name\":\"Morten Light\",\"theme\":\"light\",\"styles\":\"norm=bg:f,bgo:1\"},{\"name\":\"Francisco Goya (1)\",\"theme\":\"dark\",\"styles\":\"asgn=fg:ff0 com=fg:b,I cur=bc:f00 dfn2=fg:eb4 dfn3=fg:c79 dfn4=fg:cd0 dfn5=fg:a0d dfn=fg:a7b diam=fg:ff0 err=fg:f00,bg:822,bgo:0.5,B fn=fg:0f0 idm=fg:0f0 glb=B kw=fg:aa2 lbl=U,bg:642,bgo:0.5 lnum=fg:b94,bg:010,bgo:0 mod=bg:7,bgo:0.5 mtch=fg:0,bg:ff8,bgo:0.75 norm=fg:9c7,bg:0,bgo:1 num=fg:a8b op1=fg:d95 op2=fg:fd6 sel=bg:048,bgo:0.5 semi=fg:8 sqbr=fg:8 srch=bg:b96,bgo:0.75,fg:0 str=fg:dae tc=bg:1,bgo:1 tcpe=bg:2,bgo:1 zld=fg:d9f,B scmd=fg:0ff ucmd=fg:f80,B vtip=bg:733,fg:ff0,bgo:1,bc:900 vtt=bc:f80\"}]",
    "connectOnQuit":"1",
    "floating":"1",
    "kbdLocale":"da_DK",
    "keys":"{\"DMN\":[\"Shift-Alt-N\"],\"DMP\":[\"Shift-Alt-P\"],\"DMR\":[\"Shift-Alt-D\"]}",
    "minimapShowSlider":"always",
    "prefixKey":"<",
    "selectedExe":"C:\\Program Files\\Dyalog\\Dyalog APL-64 16.0 Unicode\\dyalog.exe",
    "wse":"1",
    "zoom":"1"
  }

Interpreter:
  Version: unknown
  Platform: unknown
  Edition: unknown
  Date: unknown
```

**Comment by e9gille** [comment]
I can't replicate it. This was fixed in ac65a69
Can you please check you have the latest version.

**Comment by e9gille** [comment]
Closing, @mkromberg please re-open if still an issue
